tests/ert/unit_tests/config/test_ert_config.py

```diff
@@ -324,7 +315,7 @@ def run(self, *args):
     Path("plugin.py").write_text(
         dedent(
             """
-            from ert.plugins import ErtPlugin
+            from ert.config import ErtPlugin
```

Comment:
This change should be part of the previous commit. Please make sure the tests pass on all commits.